functions-methods.js

```diff
@@ -33,4 +54,19 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity (emailAddress) {
+    const hasAmpersand = emailAddress.includes('@');
+    const hasComma = emailAddress.includes(',');
+    const lastChar = emailAddress[emailAddress.length - 1];
+
+    if (hasAmpersand && !(hasComma) && (lastChar !== '.')) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Je kunt de uitkomst van de expressie meteen teruggeven:
```suggestion
    return hasAmpersand && !hasComma && lastChar !== '.';
```